changelog/changes/2022-06-14-ignition-rmcfg-vmware.md

```diff
@@ -0,0 +1 @@
+- VMWare: Added `ignition-delete-config.service` to remove Ignition config from VM metadata. Also see: https://coreos.github.io/ignition/operator-notes/#automatic-config-deletion
```

Comment:
```suggestion
- VMWare: Added `ignition-delete-config.service` to remove Ignition config from VM metadata, see also [here](https://coreos.github.io/ignition/operator-notes/#automatic-config-deletion)
```
Without the markdown formatting I'm not sure if it becomes a clickable link in the homepage?